### ISSUE TITLE
CLI-like output

### ISSUE THREAD

**Issue by vladdeSV** [vladdeSV/scone#67]
Enable colors with `std.stdio`'s `write` family.

I tried implementing a feature like this once by overriding the `toString()` method for the `TextStyle` struct, which outputs as an ANSI escape code which sets the colors. This functionality, however, does not belong there.

One approach would be to add a function with the signature

```d
string cli(TextStyle style);
```

where you could do the following in the standard `writeln`

```d
writeln("Welcome ", TextStyle().bg(Color.yellow).cli(), "user", TextStyle().cli(), ".");

// alternatively
auto info = TextStyle().bg(Color.yellow).cli();
auto reset = TextStyle().fg(Color.initial).bg(Color.initial).cli();
writeln("Welcome ", info, "user", reset, ".");
```

**Comment by vladdeSV** [comment]
Alternatively, another approach could be to add completely new function, for example `scone.writeln(...)` which converts each argument which is a `TextStyle` to a string and calls the appropriate `write` function within itself.

**Comment by vladdeSV** [comment]
This would mean I could deprecate my own small project [termcolor-d](https://github.com/vladdeSV/termcolor-d).

**Comment by vladdeSV** [comment]
This was implemented in a3896c73fd4ca8753486f35f7e0ad3267a473e5e.

**Comment by vladdeSV** [comment]
Windows Terminal can handle RGB color output, but RGB in colors are disabled in `Color`.

This causes some conflict, and might be related to #66.
